.changes/next-release/enhancement-Argumentprocessing-46505.json

```diff
@@ -0,0 +1,5 @@
+{
+  "category": "Argument processing",
+  "type": "enhancement",
+  "description": "Added cli_follow_urlparam option in the config file which can be set to false to disable the automatic following of string parameters prefixed with http:// or https://"
```

Comment:
1. Reference existing issues that this will close
2. We talked about creating a tracking issue where people could discuss this feature.  Did you create one?  If so, link that here as well.